lib/drab/commander.ex

```diff
@@ -169,6 +178,14 @@ defmodule Drab.Commander do
   Launched every time client browser connects to the server, including reconnects after server
   crash, network broken etc
 
+  #### `onload_init`
+  Launched synchronously just before the `onload` callback. Unlike all the others callbacks
+  that runs in their own process, this callback runs in the main process, and therefore is useful
+  to call those operations who needs the main process PID to operate correctly, 
```

Comment:
No it is not useful for that because you can just `Drab.pid(socket)` to get the pid, and this one works in any callback, not just on init.